libfreerdp/utils/smartcard_call.c

```diff
@@ -64,6 +67,9 @@ struct s_scard_call_context
 #if defined(WITH_SMARTCARD_EMULATE)
 	SmartcardEmulationContext* emulation;
 #endif
+	HANDLE hWinSCardLibrary;
+	SCardApiFunctionTable WinSCardApi;
+	PSCardApiFunctionTable pWinSCardApi;
```

Comment:
should be `const`

Reply:
maybe I'm holding it wrong... how do I make pWinSCardApi const without causing these kinds of problems:

```
FreeRDP/libfreerdp/utils/smartcard_call.c: In function ‘smartcard_call_context_new’:
FreeRDP/libfreerdp/utils/smartcard_call.c:1876:22: error: assignment of read-only member ‘pWinSCardApi’
 1876 |    ctx->pWinSCardApi = &ctx->WinSCardApi;
      |                      ^
FreeRDP/libfreerdp/utils/smartcard_call.c:1881:22: error: assignment of read-only member ‘pWinSCardApi’
 1881 |    ctx->pWinSCardApi = WinPR_GetSCardApiFunctionTable();
      |                      ^
FreeRDP/libfreerdp/utils/smartcard_call.c: In function ‘smartcard_call_context_free’:
FreeRDP/libfreerdp/utils/smartcard_call.c:1936:20: error: assignment of read-only member ‘pWinSCardApi’
 1936 |  ctx->pWinSCardApi = NULL;
```

Reply:
I've used `SCardApiFunctionTable const*` instead of `const PSCardApiFunctionTable`, which I understand should give a pointer to a const SCardApiFunctionTable, rather than a const pointer to a non-const SCardApiFunctionTable. I'm hoping this is what you meant by 'const' here